lib/Provider/AtLoginProvider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types = 1);
+
+namespace OCA\TwoFactorEmail\Provider;
+
+use OCP\Authentication\TwoFactorAuth\ILoginSetupProvider;
+use OCP\Template;
+use OCP\IUser;
+use OCA\TwoFactorEmail\Service\Email as EmailService;
+use OCP\IConfig as OCCONFIG;
+use OCP\Authentication\TwoFactorAuth\IRegistry;
+
+class AtLoginProvider implements ILoginSetupProvider {
+
+	public function __construct(IUser $myUser, $mySecret, EmailService $EmailService, OCCONFIG $occonfig, IRegistry $registry, Email $provider) {
+		$this->myUser = $myUser;
+		$this->mySecret = $mySecret;
+		$this->EmailService = $EmailService;
+		$this->occonfig = $occonfig;
+		$this->registry = $registry;
+		$this->provider = $provider;
+	}
+
+	public function setEnabledActivity() {
+        // With OCP\IConfig i can get all configured value in config/config.php so i can know if twofactor is forced
+        // And it's not in database
+		$state2fa = $this->occonfig->getSystemValue('twofactor_enforced');
```

Comment:
Since "state" is associated with the internal state of twofactor_email, $state2fa seems misleading. I propose to use "$isEnforced" instead.